__tests__/components/forecast-summary.test.jsx

```diff
@@ -1,52 +1,27 @@
 import React from 'react';
 import Enzyme from 'enzyme';
 import ForecastSummary from '../../src/components/forecast-summary';
-import WeatherIcon from 'react-icons-weather';
 
-it('renders the date', () => {
+describe('Forecast Summary', () => {
   const wrapper = Enzyme.shallow((
     <ForecastSummary
       date={123}
       temperature={25}
       description="mockDescription"
-      icon="mockIcon"
+      icon="800"
+      onSelect={jest.fn()}
     />
   ));
-  expect(wrapper.find('.forecast-summary__date').text()).toEqual('Thu 1st January');
-});
-
-it('renders the temperature', () => {
```

Comment:
You can still use `beforeEach()` in these tests. I would save you declaring the same wrapper each time.